kitsune/wiki/views.py

```diff
@@ -1445,3 +1473,116 @@ def what_links_here(request, document_slug):
     }
 
     return render(request, 'wiki/what_links_here.html', c)
+
+
+def get_fallback_locale(doc, request):
+    """Get best fallback local based on locale Mapping"""
+
+    # Get the Translated locales which have current revision
+    translated_locales = (doc.translations.all()
+                                          .values_list('locale', flat=True)
+                                          .exclude(current_revision=None))
+    # Get the Accepted_language header from user request
+    browser_language_code = request.META.get('HTTP_ACCEPT_LANGUAGE', None)
+    if browser_language_code is not None:
+        header_locales = parse_accept_lang_header(browser_language_code)
+    else:
+        header_locales = []
+    # Get the Locale mapping from setting
+    NON_SUPPORTED_LOCALES = settings.NON_SUPPORTED_LOCALES
+
+    # Check if the document is translated to any of the locale listed in Accepted_Language Header
```

Comment:
# Check if the document is localized into any of the locales listed in the ACCEPT_LANGUAGE header